sql/moz-fx-data-shared-prod/udf/geo_struct_unknown/udf.sql

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Instead of creating another UDF that looks very similar to what we already have, would it make sense to perhaps update:
https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/udf/geo_struct/udf.sql

to not return nulls but `??` instead if this is the preferred behaviour for derived tables?

Reply:
no because that udf is used other places and also i want to use that udf at the end to null it again

Reply:
also I had a look if I could do a flag but in BigQuery udfs you can't have parameters with default values ...

Reply:
ok, my only comment then is. Could we come up with a different, more descriptive name? Maybe something like `geo_struct_normalize` or `geo_struct_set_defaults`? If you have other ideas, I'm open to it. I'm just not sure if geo_struct`_unknown` is a good name here.